docs/competition/teams/photo-studio.md

```diff
@@ -0,0 +1,32 @@
+# Photo studio manager
+
+The role of the photo studio manager is to take nice photos of teams with their robots, and a photo of robot on their own.
+
+You should have:
+- A camera on a tripod
+- A table for the robot
+- A logbook to annotate what photos go to what team
+- A set of photo flashes
+- A photo backdrop
+
+Take many photos, and make sure the photos are in focus and correctly framed before dismissing the team.
+
+Take photos of:
+- The team posing with their robot
+- The robot itself
+
+Make sure you take note of which photos go to which team!
+
+Do this by doing both of the following:
+- Write down the ID of the photos for each team (for the first and last photos).
+- Take a photo of a piece of paper with the team's TLA before each set of team photos.
+
+# Photo studio helper
+
+The helper's job is assist the studio photographer, and to fetch teams to be photographed by the Photo studio manager. It's almost guaranteed that not all teams will want to be photographed immediately, so try to do it in order, but all teams must eventually be photographed!
+
+Make sure the teams bring their robots, and try to maintain a queue of max 2 teams to be photographed. Any more will be a waste of teams' time.
+
+You should have:
+- A clipboard with a checklist of teams.
+- A map of the venue.
```

Comment:
Why?

Reply:
To find the teams

Reply:
Is it really the photo studio persons job to do this? That means whilst they're hunting for teams, if someone comes over, there won't be anyone there

Reply:
the photographer will still be there it is just the assistant doing the fetching,

Reply:
ah, i see, i forgot there were 2 roles for that. In that case yeah this makes sense :+1:

Reply:
There's 2 roles for the photo studio, the photographer, and the helper. The helper is the one who grabs teams and the photographer takes the photos